ngx-tools/type-guards/src/is-keyboard-event/is-keyboard-event.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { KEYS } from '@terminus/ngx-tools/keycodes';
+import {
+  createKeyboardEvent,
+  createMouseEvent,
+} from '@terminus/ngx-tools/testing';
+
+import { isKeyboardEvent } from './is-keyboard-event';
+
+
+describe(`isKeyboardEvent`, function() {
+
+  test(`should return true for a keyboard event`, function() {
+    const event = createKeyboardEvent('keyup', KEYS.ENTER);
+    expect(isKeyboardEvent(event)).toEqual(true);
+  });
+
+
```

Comment:
These tests don't seem to cover as much ground as some of the others. What about a keydown, keypress? Any other values from KEYS?

Same for the negative-test

Reply:
It seems like that would just be testing the creation of other events. We're at 100% line and path coverage for all of these.